### DISCUSSION TITLE
Plans for a new release?

### DISCUSSION THREAD

**Discussion by bpuertolas** [cyrilgdn/terraform-provider-postgresql#338] [General]
Hello,

I would like to use the "SECURITY OF DEFINER" feature in my function. It seems that it has been implemented here: https://github.com/cyrilgdn/terraform-provider-postgresql/pull/307

Are there any plans for a new release of the provider?

**Comment by talbx** [comment]
@bpuertolas this feature was already merged and included in [release v1.21.0](https://github.com/cyrilgdn/terraform-provider-postgresql/releases/tag/v1.21.0)
